websites/C/Coursera/presence.ts

```diff
@@ -0,0 +1,350 @@
+type Functionlize<T> = {
+	[P in keyof T]: () => T[P];
+};
+
+interface Route extends Functionlize<Partial<PresenceData>> {
+	path: RegExp;
+	playback?(): boolean;
+	run?(): PresenceData;
+}
+
+const enum LoAssets {
```

Comment:
```suggestion
const enum Assets {
```

Reply:
> ```suggestion
> const enum Assets {
> ```

won't this inflict with the default assest in presences?

Reply:
no.